Crossroads ship

CadenceArgent opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->


## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
Adds the PHI Crossroads shuttle. This is a civilian shuttle meant to act as a hub of sorts, allowing multiple more specialized ships to dock and provide services for the greater whole. It provides uncluttered walkways, cozy meeting areas, and pretty scenery including grass and a public park.

This ship is intended to fill a role akin to that of the Bazaar, having multiple docking ports, spacious interior for character actions, and amenities helpful to operation of such a vessel including a janitor's closet, dorms for management staff, and a reception desk.

It comes equipped with ample fire and atmos safety measures, including multiple fire alarms, segmented firelocks, warning lights, and emergency safety devices.

Flight control is average. It accelerates at an average speed, a bit faster accelerating forward. Turning is not incredibly responsive, the Crossroad's bulk shows with this part of its maneuverability. However, this is not necessarily a detriment, as the ship is meant to stay stationary or work in tandem with other vessels.

Key numbers:
Crew - 1-3
Tile count - 528 (25W x 28L) (Medium-Large)
Price - 74000 (appraises at around 59k but has been marked up in compliance with the Tier system)

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
With recent changes to how trade outposts and depots work, docking space has been very taxed. This could allow for additional ports to be added by players, such that one Trade Outpost docking port could become three.

It also serves the purpose of connecting players with a comfortable and inviting environment to park their ships and engage in casual character RP. It encourages players to fly together, help each other, and forego the usual lone wolf mindset that many players seem to have.

## Technical details
<!-- Summary of code changes for easier review. -->
Adds two .yml files, one with the grid itself and the other with the shipyard prototype information.

## How to test
<!-- Describe a procedure to test this feature, along with expected output/behavior. -->
Checkout the branch, build the project, and purchase the ship from the civilian shipyard console.

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
Initialized Crossroads:
![Initialized](https://github.com/user-attachments/assets/16d5bc54-dd60-4372-b839-d20ef54ee3f0)
Uninitialized Crossroads:
![Uninitialized](https://github.com/user-attachments/assets/fce8d953-5454-4457-9f4b-17de3201ac1f)
Mass Signature:
![Mass Signature](https://github.com/user-attachments/assets/ff23fe20-b90d-4b21-b2be-980b85df9fbb)

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [ x ] I have read [CONTRIBUTING.md](https://github.com/new-frontiers-14/frontier-station-14/blob/master/CONTRIBUTING.md) and and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ x ] I have added media to this PR or it does not require an ingame showcase.
- [ x ] I have reviewed the [Ship Submission Guidelines](https://frontierstation.wiki.gg/wiki/Ship_Submission_Guidelines) if relevant.
- [ x ] I confirm that AI tools were not used in generating the material in this PR.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them. -->
None that I could find!

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
Major Additions:
- Initial upload of the PHI Crossroads shuttle, and shipyard entry.

Minor Fixes:
- Updated name in shipyard prototype file to not overwrite the Eagle.
- Changed price of shuttle to be compliant with tier system.
- Updated search tags in shipyard prototype to Civilian.
- Set guidebook entry to Null, guidebook to be added at a later time.

dvir001 commented:
Please fix the PR title to the same format of other ships PR's, please add a ship guidebook.

CadenceArgent commented:
Gotcha, updated the title, working on the guidebook entry now

CadenceArgent commented:
Added guidebook entry, brought title and format in line with other shuttle pull requests, even began work on a map image for the guidebook

AF-Buyrcsp2 commented:
i think thats a wrong usage of changelog?
i think that section is used for the server changelog rather than the ship

CadenceArgent commented:
The lights on the dashboard of the bridge are detailed in the guide, they were meant as indicators for the various states (anchor enabled/disabled, thrusters enabled/disabled) but they have been removed to comply with standards. When first uploaded, anchors were not able to be linked, so I am now adding the second button inside the cockpit for the anchor toggle. There really isnt anywhere else to put it unfortunately, so I was waiting for the link-able anchor PR which seems to be fixed now. 

Cleared up main throughlines in the theatre, still cluttered but not uncomfortably. Moved vending machines into the cargo area, added shutters and buttons for the shutters in each of the shopfronts, removed the second pilot chair from the bridge, adjusted the bay windows in the south half to be better for foot traffic

whatston3 commented on Resources/Prototypes/_NF/Shipyard/crossroads.yml:
```diff
@@ -0,0 +1,40 @@
+            prefixCreator: '14'
+        - type: StationJobs
+          availableJobs:
+            Contractor: [ 0, 0 ]
+            Pilot: [ 0, 0 ]
+            Mercenary: [ 0, 0 ]
```
```suggestion
            ContractorInterview: [ 0, 0 ]
            PilotInterview: [ 0, 0 ]
            MercenaryInterview: [ 0, 0 ]
```
Use new interview jobs.

CadenceArgent replied:
This will be fixed in the next commit, thanks for letting me know!
